packages/comparator/src/type.ts

```diff
@@ -1,6 +1,9 @@
 // Copyright 2022 Touca, Inc. Subject to Apache-2.0 License.
 
+import { Buffer } from 'node:buffer'
 import { Decimal } from 'decimal.js'
+import { diff_match_patch } from 'diff-match-patch'
+import * as jsonDiff from 'json-diff'
```

Comment:
Looks like we're only using `diff` and `diffString`:
```suggestion
import { diff, diffString } from 'json-diff'
```